Templates/Swift/Sources/Request.swift

```diff
@@ -2,7 +2,7 @@
 
 import Foundation
 
-extension {{ options.name }}{% if tag %}.{{ options.tagPrefix }}{{ tag|upperCamelCase }}{{ options.tagSuffix }}{% endif %} {
+extension {% if tag %}{{ options.tagPrefix }}{{ tag|upperCamelCase }}{{ options.tagSuffix }}{% else %}Operation{% endif %} {
```

Comment:
```suggestion
extension Config{% if tag %}.{{ options.tagPrefix }}{{ tag|upperCamelCase }}{{ options.tagSuffix }}{% else %}Operation{% endif %} {
```

Reply:
No, operations do not need to be under Config, they leave on the top level.